### PR TITLE
feat: Remove funding function in Backer

### PR DESCRIPTION
**Pull request by Sotatek-Patrick-Vu**
Remove funding function and funding address in the Cardano backer

### CODE REVIEW

File: src/backer/cardaning.py [hunk 1]
```diff
@@ -49,24 +49,22 @@ class Cardano:
...
         self.client = ogmios.Client()
         self.tipHeight = 0
 
-        # retrieve backer private key and derive cardano address
-        backer_private_key = ks.pris.get(hab.kever.prefixer.qb64).raw
-        self.payment_signing_key = pycardano.PaymentSigningKey(backer_private_key, "PaymentSigningKeyShelley_ed25519", "PaymentSigningKeyShelley_ed25519")
-        payment_verification_key = pycardano.PaymentVerificationKey.from_signing_key(self.payment_signing_key)
-        self.spending_addr = pycardano.Address(payment_part=payment_verification_key.hash(),staking_part=None, network=NETWORK)
-        print("Cardano Backer Address:", self.spending_addr.encode())
+        self.payment_signing_key = pycardano.PaymentSigningKey.from_cbor(os.environ.get('FUNDING_ADDRESS_CBORHEX'))
```
Comment (iFergal) [inline src/backer/cardaning.py:57]: Lets change the env to `WALLET_ADDRESS_CBORHEX` throughout the repo. And update any variable names here. It's no longer the funding address, it's now the spending address

File: src/backer/cardaning.py [hunk 2]
```diff
@@ -49,24 +49,22 @@ class Cardano:
...
+        funding_payment_verification_key = pycardano.PaymentVerificationKey.from_signing_key(self.payment_signing_key)
+        self.funding_addr = pycardano.Address(funding_payment_verification_key.hash(), None, network=NETWORK)
+        print("Cardano Backer Spending Address:", self.funding_addr.encode())
 
         # check address balance and try to fund if necesary
```
Comment (iFergal) [inline src/backer/cardaning.py:62]: remove this comment

File: src/backer/cardaning.py [hunk 3]
```diff
@@ -49,24 +49,22 @@ class Cardano:
...
-        balance = self.getaddressBalance(self.spending_addr.encode())
+        balance = self.getaddressBalance(self.funding_addr.encode())
         if balance and balance > MINIMUN_BALANCE:
             print("Address balance:", balance/1000000, "ADA")
         else:
-            self.fundAddress(self.spending_addr)
+            print("Address balance:", "n/a", "ADA")
```
Comment (iFergal) [inline src/backer/cardaning.py:67]: we should display an error that the wallet is empty, and exit

File: scripts/generate_funding_cborhex_cardano.py
Comment (iFergal) [inline scripts/generate_funding_cborhex_cardano.py:1]: Why is this removed? It can just be renamed

Comment (Sotatek-Patrick-Vu) [reply scripts/generate_funding_cborhex_cardano.py:1]: I thouht that we don't use this any more
I push it back and change script/variables name

**Review by iFergal — APPROVED** [review]
